gpii/node_modules/matchMakerFramework/src/MatchMakerFramework.js

```diff
@@ -64,6 +64,10 @@
      *    userToken, preferences, deviceContext, solutionsRegistryEntry
      */
     gpii.matchMakerFramework.preProcess = function (that, initialPayload) {
+        // merge the OS solutions registry with the web-based solutions registry:
+        initialPayload.solutionsRegistryEntries = $.extend({},
```

Comment:
We can't have this stuck into the MatchMakerFramework base utilities like this - it needs to be a feature of a particular kind of FlowManager on a particular platform. Also, modifying the input payload mustn't happen